docs/getting-started/datapackage-find-prepare-share-guide.md

```diff
@@ -4,19 +4,15 @@ date: 2018-04-26
 authors: Dmytro Gierman, Branko Djordjevic
 ---
 
-**In this guide, we describe a way to gather information and make it presentable and easy to use for your customers.**
-
-[dp]: https://datahub.io/docs/data-packages
-
-We'll introduce the concept of [Data Packages][dp] and walkthrough how to get data manually or using automated processes, clean it up, create a data package and finally share it online using datahub.io.
+**In this guide we walk through creating a data package from scratch and publishing to the DataHub. As a bonus, we'll also show you how to collect data and clean up before packing it for publication.**
 
 [[toc]]
 
 # What is Data Package
+[csv]: /docs/data-packages/csv
```

Comment:
Spacing after heading!

Also [csv] link should be after the point it is used usually.